app/models/planning_application.rb

```diff
@@ -279,7 +279,17 @@ def result_present?
   end
 
   def validation_requests
-    (description_change_validation_requests + replacement_document_validation_requests + additional_document_validation_requests + other_change_validation_requests + red_line_boundary_change_validation_requests).sort_by(&:created_at).reverse
+    (replacement_document_validation_requests + additional_document_validation_requests + other_change_validation_requests + red_line_boundary_change_validation_requests).sort_by(&:created_at).reverse
+  end
+
+  def open_description_change_requests
+    description_change_validation_requests.select(&:open?)
+  end
+
+  def rejected_description_change
+    description_change_validation_requests.select do |request|
+      request&.approved == false
```

Comment:
why would the request be nil?

Reply:
so here you're selecting the latest request that is not approved?

Reply:
yes, I'm selecting the last request that was not approved because we want to display rejected description changes on the "new" page whenever a new request is created - so if an applicant has rejected a description change or a request was cancelled, that will be visible on the new page